src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -55,19 +55,24 @@
                                TargetMonikers="@(TestNugetTargetMoniker)">
       <Output TaskParameter="ResolvedCopyLocalItems" ItemName="TestCopyLocal" />
     </PrereleaseResolveNuGetPackageAssets>
-
+    
     <ItemGroup Condition="'$(TestWithCore)' != 'true'">
-
       <_TestCopyLocalXunitFiles Include="$(PackagesDir)xunit.runner.console\2.1.0\tools\*.exe" />
       <_TestCopyLocalXunitFiles Include="$(PackagesDir)xunit.runner.console\2.1.0\tools\*.dll" />
-      <_TestCopyLocalXunitFiles Condition="'$(TestAppX)' == 'true'" Include="$(PackagesDir)microsoft.xunit.runner.uwp\**\*.*" />
-
       <TestCopyLocal Include="@(_TestCopyLocalXunitFiles)" >
         <NuGetPackageId>%(FileName)</NuGetPackageId>
         <NuGetPackageVersion>2.1.0</NuGetPackageVersion>
         <Private>false</Private>
       </TestCopyLocal>
     </ItemGroup>
+    
+    <ItemGroup Condition="'$(TestAppX)' == 'true'">
+      <_TestCopyLocalAppXRunner Include="$(PackagesDir)microsoft.xunit.runner.uwp\$(AppXRunnerVersion)\**\*.*"/>
+      <TestCopyLocal Include="@(_TestCopyLocalAppXRunner)">
+        <NuGetPackageId>%(FileName)</NuGetPackageId>
+        <SourcePath>%(Identity)</SourcePath>
```

Comment:
What does `%(Identity)` get set to here? I would expect it to be a full file path.